src/components/Accordion/AccordionSection/AccordionSection.test.tsx

```diff
@@ -27,16 +28,17 @@ describe("AccordionSection ", () => {
         titleElement="h4"
       />
     );
-
-    expect(wrapper.find("h4.p-accordion__heading")).toHaveLength(1);
-    expect(wrapper.find("h4").first().prop("aria-level")).toBeNull();
-    expect(wrapper).toMatchSnapshot();
+    // Query for the specific element as defined in the titleElement prop.
+    // eslint-disable-next-line testing-library/no-node-access
+    const title = document.querySelector("h4");
```

Comment:
Similar comment as above. We should query the title element (semantic way) and check if it is h4.